fuse/utils/tests/test_ndict.py

```diff
@@ -246,7 +246,15 @@ def test_print_tree(self) -> None:
         ndict["a.d"] = 23
         ndict["e"] = 10
 
-        ndict.print_tree(print_values=True)
+        gts = [
```

Comment:
All four possible permutations of the result (I think it isn't necessarily ordered)